Dockerfile

```diff
@@ -1,24 +1,18 @@
-FROM ubuntu:16.04
+FROM python:3.6
```

Comment:
I got this to work with the following:

```
FROM ubuntu:16.04

RUN apt-get update && apt-get install -y python3 python3-pip python3-wheel python3-six python3-pip

RUN mkdir /dk
COPY . /dk
WORKDIR /dk
RUN python3 -m pip install -r requirements.txt

CMD ["/bin/bash"]
```

And then:

```
docker run draft-kings-fun-fork nosetests
```

passes tests. I have some changes queued up with this and some Travis stuff, so I can PR onto your branch.